SpecialOperations2.Altis/sos/logistics/SOS_LogisticsSpawn.hpp

```diff
@@ -0,0 +1,80 @@
+class SOS_LogisticsSpawn {    
+    class AmmoDrop {
```

Comment:
Rename to **AmmoCrate**

And can you nerf this crate so it just contains ammo.